jre/javatests/com/google/j2cl/jre/java9/util/stream/CollectorsTest.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2024 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.j2cl.jre.java9.util.stream;
+
+import static com.google.j2cl.jre.java8.util.stream.CollectorsTest.applyItems;
```

Comment:
Yikes, we don't allow this.
Let's move new tests to java8 even though this is a new API.